source/Xunit.Gherkin.Quick/Feature.cs

```diff
@@ -45,13 +45,27 @@ internal void Scenario(string scenarioName)
                 object[] methodParamValues = null;
                 if (methodParams.Length > 0)
                 {
-                    var methodParamStringValues = stepRegexMatch.Groups.Cast<Group>().Skip(1).Select(g => g.Value).ToList();
+                    var methodParamStringValues = stepRegexMatch.Groups.Cast<Group>().Skip(1).Select(g => g.Value).ToList<object>();
 
-                    if (methodParamStringValues.Count < methodParams.Length)
+                    if (methodParamStringValues.Count < methodParams.Length && parsedStep.Argument == null)
+                    {
                         throw new InvalidOperationException($"Method `{matchingStepMethod.method.Name}` for step `{parsedStep.Keyword}{parsedStep.Text}` is expecting {methodParams.Length} params, but only {methodParamStringValues.Count} param values were supplied.");
+                    }
+
+                    // A step argument (Table/Multiline String, etc.) is always last is the list of arguments
```

Comment:
Is that true? Can it be not last as well?

Reply:
It must be the last argument.  I can remove the comment.

Reply:
Actually I was thinking it could be at any index. Does it make sense to have it at other position, or the last only makes sense?
I don't mind about the comment.